commands/polls/vote.js

```diff
@@ -0,0 +1,107 @@
+const { Command } = require('discord.js-commando');
+const { RichEmbed } = require('discord.js');
+const oneLine = require('common-tags').oneLine;
+
+module.exports = class VoteCommand extends Command {
+    constructor(client) {
+        super(client, {
+            name: 'vote',
+            group: 'polls',
+            memberName: 'vote',
+            description: "Starts a yes/no/don't care vote.",
+            examples: ['!vote "Do you like to vote?" "I mean who doesn\'t right?!" 5'],
+            args: [
+                {
+                    key: 'question',
+                    prompt: 'What is the vote question?',
+                    type: 'string',
+                    validate: question => {
+                        if (question.length < 101 && question.length > 11) return true;
```

Comment:
Same as with the other poll, I see no reason to place a minimum size on the question.

Reply:
fixed